M2/Macaulay2/packages/VirtualResolutions/doc.m2

```diff
@@ -0,0 +1,549 @@
+     While graded minimal free resolutions are useful for studying quasicoherent
+     sheaves over projective space, when working over a product of projective spaces or, more generally,
+     over smooth projective toric varieties, graded minimal free resolutions over the Cox ring
+     seem too restricted by algebraic structure that is in some sense unimportant geometrically. By allowing
+     a limited amount of homology, virtual resolutions offer a more flexible alternative for
+     studying toric subvarieties when compared to minimal graded free resolutions.
+
+     Introduced by Berkesch, Erman, and Smith in {\em Virtual resolutions for a product of projective spaces}
+     (see @{HREF("http://arxiv.org/abs/1703.07631","arXiv:1703.07631")}@) if $X$ is a smooth toric variety, $S$ the Cox ring of $X$
+     graded by the Picard group of $X$, and $B\subset S$ the irrelevant ideal of $X$, then
+     a virtual resolution of a graded $S$-module $M$ is a complex of graded free $S$-modules, which
+     sheafifies to a resolution of the associated sheaf of $M$.
+
+     This package provides tools for constructing and studying virtual resolutions for products of projective spaces.
+     In particular, it implements a number of the methods for constructing virtual resolutions for products of projective
+     spaces as introduced by Berkesch, Erman, and Smith. This package also contains methods for constructing curves in
+     $\mathbb{P}^1\times\mathbb{P}^2$, as these are a natural source for interesting virtual resolutions.
+
+     As a running example, consider three points $([1:1],[1:4])$, $([1:2],[1:5])$, and $([1:3],[1:6])$
+     in $\mathbb{P}^1 \times \mathbb{P}^1$.
+
+    Example
+     X = toricProjectiveSpace(1)**toricProjectiveSpace(1);
+     S = ring X;
+     B = ideal X;
+     J = saturate(intersect(
+         ideal(x_1 - x_0, x_3 - 4*x_2),
+         ideal(x_1 - 2*x_0, x_3 - 5*x_2),
+         ideal(x_1 - 3*x_0, x_3 - 6*x_2)), B);
+     minres = res J;
+     multigraded betti minres
+    Text
+     As described in Algorithm 3.4 of Berkesch, Erman, and Smith's
+     paper, one may construct a virtual resolution of a module from its graded minimal free resolution and
+     an element of the multigraded Castelnuovo-Mumford regularity of the module. (See Maclagan and Smith's paper
+     {\em Multigraded Castelnuovo-Mumford Regularity} (@{HREF("http://arxiv.org/abs/math/0305214","arXiv:math/0305214")}@) for the definition of multigraded regularity.)
+     Building on the TateOnProducts package, this package contains a function allowing one
+     to compute the minimal elements of the multigraded Castelnuovo-Mumford regularity of a $B$-saturated module.
+
+     Continuing the example from above, we see that $(2,0)$ is an element of the multigraded
+     regularity of $S/J$. From this we can compute a virtual resolution of $S/J$.
+    Example
+     multigradedRegularity(X, J)
+     vres = virtualOfPair(J, {{3,1}})
+     multigraded betti vres
```

Comment:
`multigraded` is undefined, so this example fails, even though #906 has been merge.

Reply:
I can't reproduce the issue, but since there isn't much time I removed `multigraded` for now.